packages/api/emails/verifyHcEmail/html.pug

```diff
@@ -1,13 +1,14 @@
 div.main
-  img(src=`${appURL}/email/header.jpg` alt='header' style='width:100%; height:auto;')
+  img(src=`${process.env.APP_URL}/email/header.jpg` alt='header' style='width:100%; height:auto;')
```

Comment:
Is this change related to what this PR is supposed to do? https://github.com/the-difference-engine/ksf/pull/202

Reply:
This is getting taken care of in another PR so this can probably be undone.